R/utils.R

```diff
@@ -324,6 +324,145 @@ date_to_numeric_modulus <- function(dt, date_column, timestep) {
   return(dt[])
 }
 
+#' Set caching location for Stan models
+#'
+#' This function allows the user to set a cache location for
+#' Stan models rather than a temp directory. This can reduce the
+#' need for model compilation on every new model run across sessions.
+#'
+#' @param path A valid filepath representing the desired cache location
+#'
+#' @return The string of the filepath set
+#' @examples
+#' # Set to local directory
+#' dir.create("test")
+#' enw_set_cache("test")
+#' enw_get_cache()
+#' # Use  the package cache in R >= 4.0
+#' if(R.version.string >= 4.0) {
+#'   enw_set_cache(tools::R_user_dir(package = "epinowcast", "cache"))
+#' }
+#' @importFrom cli cli_abort cli_alert
+#' @export
+
+enw_set_cache <- function(path = NULL) {
+
+  if (is.null(path)) {
```

Comment:
NULL or missing default path?

Reply:
@pearsonca comment on this here: https://github.com/epinowcast/epinowcast/pull/416#discussion_r1436600164